Packs/SentinelOne/Integrations/SentinelOne-V2/README.md

```diff
@@ -1626,6 +1626,142 @@ Run a remote script that was uploaded to the SentinelOne Script Library.
 | SentinelOne.RunRemoteScript.affected | number | Number of entities affected by the requested operation. | 
 | SentinelOne.RunRemoteScript.parentTaskId | string | The parent task ID of the script execution task. Null in case of pending execution. | 
 
+### sentinelone-get-remote-script-task-status
+
+***
+Get remote scripts tasks using a variety of filters.
+
+#### Base Command
+
+`sentinelone-get-remote-script-task-status`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| account_ids | A comma-separated list of account IDs. Example: '225494730938493804,225494730938493915'. | Optional | 
+| computer_name_contains | Free-text filter by agent computer name (supports multiple values). | Optional | 
+| count_only | If true, only total number of items will be returned, without any of the actual objects. | Optional | 
+| created_at_gt | Created at greater than. Example: '2018-02-27T04:49:26.257525Z'. | Optional | 
+| created_at_gte | Created at greater or equal than. Example: '2018-02-27T04:49:26.257525Z'. | Optional | 
+| created_at_lt | Created at lesser than. Example: '2018-02-27T04:49:26.257525Z'. | Optional | 
+| created_at_lte | Created at lesser or equal than. Example: '2018-02-27T04:49:26.257525Z'. | Optional |
+| cursor | Cursor position returned by the last request. Use to iterate over more than 1000 items. Example: 'YWdlbnRfaWQ6NTgwMjkzODE='. | Optional |
+| description_contains | Only include tasks with specific description. | Optional |
+| detailed_status_contains | Only include tasks with specific detailed status. | Optional |
+| group_ids | List of Group IDs to filter by. Example: '225494730938493804,225494730938493915'. | Optional |
+| ids | List of IDs to filter by. Example: '225494730938493804,225494730938493915'. | Optional |
+| initiated_by_contains | Only include tasks from specific initiating user. | Optional |
+| limit | Limit number of returned items (1-1000). Example: '10'. | Optional |
+| parent_task_id | parent task id to fetch the status by. Example: '225494730938493804'. | Required |
+| parent_task_id_in | List of IDs to filter by. | Optional |
+| query | Query. | Optional |
```

Comment:
In what format should the query be?

Reply:
It will be string format

Reply:
Please update it with an example @munna-metron

Reply:
sure

Reply:
```suggestion
| query | A free-text search term that will match applicable attributes (sub-string match). | Optional |
````